### PR TITLE
Initial XC 16.0 Pages

### PR DESCRIPTION
**Pull request by pgyula**

### CODE REVIEW

File: content/stack_reports/osx-xcode-16.0.x.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Xcode 16.0
+summary: macOS stack for building with Xcode 16.0
+platform: macOS
+xcode: "16.0"
+flavor: stable
+weight: 16
```
Comment (ofalvai) [inline content/stack_reports/osx-xcode-16.0.x.md:7]: We can already set this to the correct value

Comment (pgyula) [reply content/stack_reports/osx-xcode-16.0.x.md:7]: changed flavor to stable16

Comment (ofalvai) [reply content/stack_reports/osx-xcode-16.0.x.md:7]: No, I meant the weight that controls the order on the homepage and in the sidebar.

The flavor is okay as `stable`, it's what we have today for other stacks too (see sidebar): https://stacks.bitrise.io/stack_reports/

**Review by ofalvai — APPROVED** [review]

**Comment by herculesjr** [comment]
Looks like Xcode got finally released :)

**Pull request merged**